plugin-hrm-form/src/components/tabbedForms/CSAMReportDropdown.tsx

```diff
@@ -6,21 +6,33 @@ import { StyledCSAMReportDropdown, StyledCSAMReportDropdownList, StyledCSAMRepor
 type OwnProps = {
   handleChildCSAMType: () => void;
   handleCounsellorCSAMType: () => void;
-  dropdown: boolean;
+  dropdownState: [boolean, (state: boolean) => void];
```

Comment:
```suggestion
  dropdownState: ReturnType<typeof React.useState<boolean>>;
```
might be more explicit?